### PR TITLE
Sync status

### PR DESCRIPTION
**Pull request by h00130372**

### CODE REVIEW

File: pkg/services/rds/lastoperation.go [hunk 1]
```diff
@@ -15,47 +14,44 @@ func (b *RDSBroker) LastOperation(instanceID string, operationData database.Oper
 
 	// Log opts
 	b.Logger.Debug(fmt.Sprintf("lastoperation rds instance opts: instanceID: %s operationData: %v", instanceID, models.ToJson(operationData)))
+	ids, err, getRealErr := SyncStatusWithReal(b, instanceID, operationData.ServiceID,
```
Comment (liu-sheng) [inline pkg/services/rds/lastoperation.go]: 'getRealErr' looks like a method name, maybe 'realErr' ?

Comment (kiwik) [reply pkg/services/rds/lastoperation.go]: +1, and `db_instance` is more readable than `ids`, I prefer `db_instance, db_error, service_error:= ...`

File: pkg/services/rds/utils.go [hunk 2]
```diff
@@ -45,3 +48,52 @@ func BuildBindingCredential(
 	}
 	return bc, nil
 }
+
+func SyncStatusWithReal(b *RDSBroker, instanceID string, serviceID string, planID string,
```
Comment (kiwik) [inline pkg/services/rds/utils.go]: `SyncStatusWithService` better than `SyncStatusWithReal`

**Comment by edisonxiang** [comment]
Thanks:)

**Pull request merged**